code/modules/power/supermatter/supermatter_delamination/cascade_delam_objects.dm

```diff
@@ -90,6 +90,8 @@
 	return COMPONENT_CANCEL_ATTACK_CHAIN
 
 /obj/crystal_mass/Destroy()
+	STOP_PROCESSING(SSsupermatter_cascade, src)
+	QDEL_LIST(light_sources)
```

Comment:
not a good fix.
Light sources are supposed to manage their own connection to this list. If they aren't, then that's a bug with light source code itself, or something readding a qdeleted one. it's not meant to be fixed like this